spec/furniture/marketplace/product_tags_system_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe "Product Tags", type: :system do
+  let(:space) { create(:space, :with_entrance, :with_members) }
+  let(:marketplace) { create(:marketplace, :ready_for_shopping, room: space.entrance) }
+
+  before do
+    sign_in(space.members.first, space)
+  end
+
+  scenario "Adding Tags to a Product" do # rubocop:disable RSpec/Capybara/FeatureMethods,RSpec/ExampleLength
```

Comment:
I think we should disable both of these cops in system specs. Thoughts?

Reply:
Yeah, that makes sense.